### PR TITLE
fix(landing): Trigger change event when debug value changes and disable switcher for debug mode.

### PR DESCRIPTION
**Pull request by Wentao-Kuang**

### CODE REVIEW

File: packages/landing/src/config.map.ts [hunk 1]
```diff
@@ -85,7 +85,7 @@ export class MapConfig extends Emitter<MapConfigEvents> {
     this.style = style ?? null;
     this.layerId = layerId.startsWith('im_') ? layerId.slice(3) : layerId;
     this.tileMatrix = tileMatrix;
-    this.debug = debug;
```
Comment (blacha) [inline packages/landing/src/config.map.ts:88]: can you also rename it to `isDebug` to show its a boolean

Comment (Wentao-Kuang) [reply packages/landing/src/config.map.ts:88]: Thanks fixed.

Comment (blacha) [reply packages/landing/src/config.map.ts:88]: was meaning `this.isDebug = debug` not the setter function name.

Comment (Wentao-Kuang) [reply packages/landing/src/config.map.ts:88]: Sorry, updated.

**Review by blacha — APPROVED** [review]

**Pull request merged**